tests/containers/run_container_in_k8s.pm

```diff
@@ -76,4 +79,8 @@ sub post_run_hook {
     $self->cleanup();
 }
 
+sub test_flags {
+    return {fatal => 0};
+}
```

Comment:
Why is this not fatal?

Reply:
We don't want to stop all the following tests. Think that we have 3 providers to test